pct/modules/04/power-flow.ipynb

```diff
@@ -431,9 +442,9 @@
     {
      "data": {
       "text/plain": [
-       "array([1.06      , 1.045     , 1.01      , 1.04601266, 1.05232727,\n",
-       "       1.07      , 1.07439676, 1.09      , 1.06837532, 1.0613056 ,\n",
-       "       1.06219002, 1.05612789, 1.05221949, 1.04349031])"
+       "array([1.06      , 1.045     , 1.01      , 1.01919406, 1.02024525,\n",
+       "       1.07      , 1.06202177, 1.09      , 1.05594395, 1.05095431,\n",
+       "       1.05697622, 1.05499798, 1.04999583, 1.0359623 ])"
```

Comment:
The results now are close to ANDES results (excerpted below):
array([1.06      , 1.045     , 1.01      , 1.01767085, 1.01951386,
       1.07      , 1.06151953, 1.09      , 1.05593172, 1.05098462,
       1.05690652, 1.05518856, 1.05038171, 1.03552994])